### PR TITLE
support keras functional model and tuning fallback layers

### PR DESCRIPTION
**Pull request by ClarkChin08**
## Type of Change

feature 

## Description
Enable keras functional model by recover it's graph connections structure and with custom layers.

JIRA ticket: xxx

## Expected Behavior & Potential Risk

Have Tested on mnist resnet50_v2 and xception keras model with accuracy (<1%) and performance.(1.25X ~ 2X).

## Dependency Change?

No library added.

### CODE REVIEW

File: examples/keras/xception/quantization/ptq/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+### 1. Installation
+```shell
+# Install Intel® Neural Compressor
+pip install neural-compressor
+```
+### 2. Install Intel Tensorflow
+```shell
+pip install intel-tensorflow
+```
+> Note: Supported Tensorflow [Version](../../../../../../../README.md).
```
Comment (chensuyue) [inline examples/keras/xception/quantization/ptq/README.md:19]: same issue.

File: examples/keras/xception/quantization/ptq/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Intel Extension for Tensorflow is mandatory to be installed for quantizing the model on Intel GPUs.
+
+```shell
+pip install --upgrade intel-extension-for-tensorflow[gpu]
+```
+Please refer to the [Installation Guides](https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-focal-dc.html) for latest Intel GPU driver installation.
+For any more details, please follow the procedure in [install-gpu-drivers](https://github.com/intel-innersource/frameworks.ai.infrastructure.intel-extension-for-tensorflow.intel-extension-for-tensorflow/blob/master/docs/install/install_for_gpu.md#install-gpu-drivers).
+
+#### Quantizing the model on Intel CPU(Experimental)
+Intel Extension for Tensorflow for Intel CPUs is experimental currently. It's not mandatory for quantizing the model on Intel CPUs.
```
Comment (chensuyue) [inline examples/keras/xception/quantization/ptq/README.md:32]: same question.

File: examples/.config/model_params_keras.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (chensuyue) [inline examples/.config/model_params_keras.json:1]: Please register new model info in the json file.

**Comment by chensuyue** [comment]
image
image

**Comment by chensuyue** [comment]
The performance gain is not good as expected, so we need to continue enhance this feature after this PR merged.
